streamalert/scheduled_queries/handlers/athena.py

```diff
@@ -0,0 +1,255 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import json
+import uuid
+
+from botocore.exceptions import ClientError
+
+
+class AthenaQueryExecutionError(Exception):
+    """Exception to be raised when an Athena query fails"""
+
+
+class AthenaClient:
```

Comment:
would it be possible to reuse / adapt the `AthenaClient` class we already have? it seems there's a lot of duplicative logic here